benchmarks/test/Makefile

```diff
@@ -0,0 +1,21 @@
+TEST_ARGS = $(shell echo $@ | perl -pe 's/([^.]*)\.([^.]*)\.([^.]*)\.test/--accelerator=$$1 --test=$$2 --report=$$3/')
```

Comment:
@JackCaoG , is this the way to add such a test or is there a more idiomatic way to add such tests in PyTorch/XLA?

Reply:
we run out CI in https://github.com/pytorch/xla/blob/1ea23741cadbe754728d2dac42fcfcb2718806dc/.circleci/common.sh#L128

that script will run python tests.

Reply:
Thanks! Sent https://github.com/pytorch/xla/pull/6035 as a follow-up PR